Puppetfile

```diff
@@ -28,7 +28,8 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts'
+  :git => 'https://github.com/graduway/ec2tagfacts',
+  :branch => 'windows'
```

Comment:
This should be merged to master, please open a PR. Then we will submit a PR upstream. :)